### PR TITLE
marketing footer links, styling

### PR DESCRIPTION
**Pull request by narthur**

### CODE REVIEW

File: src/containers/home.tsx [hunk 1]
```diff
@@ -386,80 +389,105 @@ export default function Home({ data }: HomeProps): JSX.Element {
...
+									<FormattedMessage
+										id={'homePage__footerDownloadAppButton'}
+										defaultMessage={'Download app'}
+										description={'home page footer download app button'}
+									/>
+								</a>
+							</li>
+							<li>
+								<a
+									href={
```
Comment (mattleff) [inline src/containers/home.tsx:468]: `target=_blank` on this one?

**Review by mattleff — APPROVED** [review]
Looks good! The font size feels small on both the footer link's headings and the links themselves, but maybe it's proportional to the font sizes for the other elements.

**Comment by narthur** [comment]
> Looks good! The font size feels small on both the footer link's headings and the links themselves, but maybe it's proportional to the font sizes for the other elements.

Maybe my eyeballing isn't accurate, this is what I was basing on:

https://www.figma.com/file/SKSajWfEkf56GQfZyI2S4a/Design-System?node-id=1075%3A0

**Comment by mattleff** [comment]
@narthur That shows a font-size for the heading of `18px` and the links of `14px`. I'm seeing `13.28px` and `11.19px` in Safari.

**Pull request merged**